.github/workflows/create-release.yml

```diff
@@ -7,80 +7,42 @@ permissions:
-        include:
-          - target: x86_64-unknown-linux-gnu
-            os: ubuntu-latest
-            arch: x86_64
-            ext: deb
-            protoc-arch: linux-x86_64
-          - target: x86_64-apple-darwin
-            os: macos-latest
-            arch: x86_64
-            ext: pkg
-            protoc-arch: osx-x86_64
-          - target: aarch64-apple-darwin
-            os: macos-latest
-            config-file: fpm/osx.fpm
-            arch: aarch64
-            ext: pkg
-            protoc-arch: osx-aarch_64
-        #- target: x86_64-pc-windows-msvc
-        #  os: windows-latest
-    runs-on: ${{ matrix.os }}
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
     env:
-      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
       - uses: actions/checkout@v3
-      - name: Install Rust
-        run: |
-          rustup update stable
-          rustup default stable
-      - name: Setup target
-        run: rustup target add ${{ matrix.target }}
-      - name: Run package script
+        with:
+          fetch-depth: 0
+      - id: release
         run: |
-          cargo build --release --locked --target ${{ matrix.target }}
-      - name: Upload artifact
-        uses: actions/upload-artifact@v3
+          export TAG=$(cargo metadata --format-version=1 --no-deps | jq -r '.packages[0].version' )
+          echo "Releasing "$TAG
+          echo "tag=${TAG}" >> "$GITHUB_OUTPUT"
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to Public ECR
+        uses: docker/login-action@v2
         with:
-          name: hermetic-driver-${{ matrix.target }}
-          path: target/${{ matrix.target }}/release/hermetic-driver
-
-  # release:
-  #   needs: [build-binaries]
-  #   runs-on: ubuntu-latest
-  #   env:
-  #     CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_TOKEN }}
-  #     GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  #   steps:
-      # - uses: actions/checkout@v3
-      #   with:
-      #     fetch-depth: 0
-      # - uses: actions/download-artifact@v3
-      #   with:
-      #     path: artifacts
-      # - name: check artifacts
-      #   run: |
-      #     ls artifacts/**/*
-      # - name: Install Rust toolchain
-      #   uses: dtolnay/rust-toolchain@stable
-      # - id: release
-      #   run: |
-      #     export TAG=$(cargo metadata --format-version=1 --no-deps | jq '.packages[0].version' | tr -d '"')
-      #     echo "Releasing "$TAG
-      #     gh release create "v${TAG}" -n "Release of ${TAG}" -t "v${TAG}" --latest artifacts/**/*.tar.gz
\ No newline at end of file
+          registry: public.ecr.aws
+          username: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          password: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        env:
+          AWS_REGION: us-east-1
+      - name: build and release docker image
+        uses: docker/build-push-action@v2
```

Comment:
Can/should we add building/pushing the driver image to a makefile directive? Would be nice to keep actual pipeline logic outside of workflows so we can run these steps anywhere, even locally.